tools/python-deps/setup.py

```diff
@@ -39,9 +45,17 @@
     name='camkes-deps',
     version='0.7.2',
```

Comment:
This version should get a minor bump, otherwise any python installation that's already installed it won't update to the version with ordered-set without `--force-reinstall`

Reply:
Yes, that would have been better. #107 does bump it, I think that'll be fine overall.

Reply:
The version on PiPY was 0.7.1 so I figured 0.7.2 would be OK.

Reply:
That makes sense, pypi won't let you overwrite old versions (at least not easily, afaik). Looks like Siwei only updated the repo, but not pypi when he bumped that version. I think the public state is fine, although maybe the repo history is not fully consistent with when 0.7.2 actually was published, but I don't think that's a problem.

Reply:
Version 0.7.2 with these changes is on PyPI now, (with the exception of the whitespace changes Gerwin pushed)

Reply:
Odd, I did a camkes-deps pip install after this PR was merged and 0.7.2 was already installed for me without ordered-set. `--force-reinstall` was required to get ordered-set. I agree that bumping to 7.3 in #107 is fine overally